### PR TITLE
Add .zenodo.json file

### PR DESCRIPTION
**Pull request by neilsjefferies**
As with spec but added authors, affiliations, ORCIDs and "specification" keyword

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "license": "cc-by-4.0",
+
+    "title": "Oxford Common File Layout - Extensions",
```
Comment (awoods) [inline .zenodo.json:4]: Should the "Extensions" word be removed?

**Review by awoods — CHANGES_REQUESTED** [review]

File: .zenodo.json (hunk 1 above)
Comment (neilsjefferies) [reply .zenodo.json]: Yes, fixed!

**Review by rosy1280 — DISMISSED** [review]

**Review by zimeon — DISMISSED** [review]

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            "name": "Morley, Julian"
+        },
+        {
+            "orcid": "0000-0002-7970-7855",
+            "affiliation": "Cornell University",
+            "name": "Warner, Simeon"
+        },
+        {
+            "orcid": "0000-0002-8318-4225",
+            "affiliation": "[Harvard University",
```
Comment (awoods) [inline .zenodo.json:31]: There seems to be a stray `[` before "Harvard".

**Review by awoods — CHANGES_REQUESTED** [review]

File: .zenodo.json (hunk 2 above)
Comment (neilsjefferies) [reply .zenodo.json]: Aargh! Fixed.

**Review by awoods — APPROVED** [review]

**Review by rosy1280 — APPROVED** [review]

**Review by zimeon — APPROVED** [review]

**Pull request merged**